CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.14.1
```

Comment:
I think it'd be better to flag this as unreleased as I imagine there's a few gem bumps that would also go out and should be part of this change log entry.